index.html

```diff
@@ -1298,10 +1281,8 @@
   goals and preferences about use of data about them.</span>
 </div>
 
-Agents should be careful, cautious, and conservative in applying data minimization to any
-[=ancillary use=], but also not burden the user with additional [=privacy
-labor=]. To that end, user agents may employ user research, solicitation of general preferences, and
-heuristics about sensitivity of data or trust in a particular context. To facilitate site
+Agents should be particularly careful, cautious, and aggresive in applying data
```

Comment:
Cautious and aggressive seem contradictory, so I'm not sure how the reader will interpret it. If what we mean is that the agent should have a tendency towards minimizing the data when it's an ancillary use, I think "conservative" covers that. "Conservative" was also removed from the introductory text at the beginning, though I'm not quite sure why. "Conservative" was in Pete's text originally, which was why we had kept it in this section.

"Particularly" implies that this agents should do this in particular, but I don't see what the more general case is that we're contrasting against. The other principles in this section and the other principles in this document seem equally relevant and important.

Reply:
Removing the privacy labor point removes the conclusion the group had that we didn't want our recommendations to lead to a shift in responsibility of the labor to the user with more prompts.